### PR TITLE
Create tutorial

### LINKED ISSUE #95: Create a tutorial (vignette)

### PR DESCRIPTION
**Pull request by DouglasMesquita**
### Link to the Issue

close #95 

### Definition of Done

Tutorial is clear and it is working

### How to test changes

Try to follow the tutorial and be sure it works

### Tasks for PR author

- [x] Test your change and ensure there is no regression
- [x] Change has a corresponding issue. ***Ensure it is linked in GitHub***
- [x] Author of the change opened a pull request and assigned a reviewer

### General policy:

- If applicable - add instructions for testing
- If there’s no issue, create it. Each issue needs to be well defined and described.
- All interaction with a user, user-facing messages, plots, reports etc. are written from the perspective of the person using or receiving it. They are understandable and helpful to this person. If a user sees an error message, there is a call to action, i.e. the user knows what to do to fix it.
- README, other documentation and code comments that we have is updated with all information related to the change.
- All code has been peer-reviewed before merging into any main branch
- All changes have been merged into the main branch we use for development.
- Continuous integration checks (linter, unit tests, integration tests) are configured and pass.
- Optional: unit tests added for all new or changed logic.
- All task requirements satisfied. If not describe it here. The reviewer is responsible to verify each aspect of the task.
- Change covers only things in task. Please create new PR if you want to fix something else.

### CODE REVIEW

File: vignettes/tutorial/how-to-measure-apps-performance.Rmd [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+
+# Simulating app versions
+
+In a regular project, you may use `git` to maintain the code. In this case, it is natural to have different app's versions in different branches/commits/releases. `shiny.benchmark` take advantage of these different `git` refs to run tests under different code versions. Add the following code to `.gitignore` to avoid problems with uncommitted files later:
+
+```git
+.Rhistory
+.Rproj.user/
+.Rproj.user
+renv/
```
[the remark this answers is not in the document]
Comment (DouglasMesquita) [reply vignettes/tutorial/how-to-measure-apps-performance.Rmd:203]: It was just to make it easy for this specific example. In the real life it is not really needed.

File: vignettes/tutorial/how-to-measure-apps-performance.Rmd [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+git commit -m "first commit"
+```
+
+Also, let's create a new branch called `feature1`:
+
+```git
+git checkout -b feature1
+```
+
+At this point, we can simulate improvement in our application. To do so, let's change `Sys.sleep` time in the server function. Replace `times <- c(10, 5, 2)` by `times <- c(5, 2.5, 1)` in `server.R` and then commit the changes.
```
[the remark this answers is not in the document]
Comment (DouglasMesquita) [reply vignettes/tutorial/how-to-measure-apps-performance.Rmd:221]: It is exactly the first row of server.R. Do you think I should add all the content or can I just add where is it in the file (first row)?

**Review by jakubnowicki — APPROVED** [review]
LGTM

File: vignettes/tutorial/how-to-measure-apps-performance.Rmd (hunk 2 above)
[the remark this answers is not in the document]
Comment (jakubnowicki) [reply vignettes/tutorial/how-to-measure-apps-performance.Rmd]: Ok, we can leave it as it is now.

**Review by jakubnowicki — APPROVED** [review]

**Pull request merged**